tools/docker-compose/Makefile

```
@@ -0,0 +1,152 @@
+  COMPOSE_FILES += -f tools/docker-compose/overrides/loki.yml
+endif
+COMPOSE_FILES += $(patsubst %,-f %,$(wildcard tools/docker-compose/_sources/overrides/execution-nodes.yml))
+COMPOSE_FILES += $(patsubst %,-f %,$(wildcard tools/docker-compose/_sources/overrides/editable-deps.yml))
+COMPOSE_FILES += $(patsubst %,-f %,$(wildcard tools/docker-compose/_sources/overrides/minikube.yml))
+
+ifneq ($(ADMIN_PASSWORD),)
+	EXTRA_SOURCES_ANSIBLE_OPTS := -e admin_password=$(ADMIN_PASSWORD) $(EXTRA_SOURCES_ANSIBLE_OPTS)
+endif
+
+.PHONY: docker-compose-sources docker-compose docker-compose-up docker-compose-down \
+	docker-compose-credential-plugins docker-compose-test docker-compose-runtest \
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)